src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml

```diff
@@ -183,6 +183,22 @@
                              Style="{StaticResource TrigUnitsRadioButtonStyle}"
                              IsChecked="{x:Bind ViewModel.TrigModeGradians, Mode=TwoWay}"/>
             </StackPanel>
+
+            <TextBlock x:Name="GraphThemeHeading"
+                       x:Uid="GraphThemeHeading"
+                       Margin="0,16,0,6"
+                       Style="{StaticResource SubTitleTextBoxStyle}"
+                       AutomationProperties.HeadingLevel="Level2"/>
+
+            <StackPanel Orientation="Vertical">
+                <RadioButton x:Name="AlwaysLightTheme"
+                             x:Uid="AlwaysLightTheme"
+                             IsChecked="{x:Bind ViewModel.IsAlwaysLightTheme, Mode=TwoWay}"/>
+                <RadioButton x:Name="MatchAppTheme"
```

Comment:
Are these x:Names ever used? If not we can remove them. Here and throughout the PR.